src/types.ts

```diff
@@ -237,11 +232,37 @@ export type Currency = {
    */
   name: string;
   /**
-   * The [[FAMILIES | family]] of the cryptocurrency
+   * array of available [[Unit | units]] for the cryptocurrency
+   */
+  units: Unit[];
+};
+
+export type CryptoCurrency = BaseCurrency & {
+  /**
+   * represents the currency type.
+   * @see {@link https://github.com/LedgerHQ/ledgerjs/blob/master/packages/cryptoassets/src/types.ts|cryptoassets types} in ledgerjs for more infos
+   */
+  type: "CryptoCurrency";
```

Comment:
Would rather create and use a CurrencyType `enum` value.

Example:

```ts
type: CurrencyType.CryptoCurrency;
```

With

```ts
enum CurrencyType {
  CryptoCurrency = "CryptoCurrency",
  ERC20TokenCurrency = "ERC20TokenCurrency",
}
```